Assets/Scripts/UI/Lists/AreaListObject.cs

```diff
@@ -28,10 +28,56 @@ public void OnToggle()
 		{
 			Controler.SelectArea(InstanceId);
 		}
+		else
+		{
+			Controler.DeselectArea(InstanceId);
+		}
 	}
 
 	public void OnRemove()
 	{
 		Controler.Remove(InstanceId);
 	}
+
+	// TODO: We need a top-level input monitor that raises events the rest of the UI can listen for like "MoveToNextField".
+	// Consider this code a QoL Prototype. We'll put something like this on each of the Vector/Color fields. Or we could do
+	// something more comprehensive - but that would mean paying closer attention to EVERY interactable element in the entire UI
```

Comment:
please limit the line length of comments to something reasonable. E.g. 80 characters

Reply:
Comment reduced